src/components/Table/TableBase.tsx

```diff
@@ -0,0 +1,678 @@
+import { faSort } from '@fortawesome/free-solid-svg-icons/faSort';
```

Comment:
Could we have a single commit w/o change type that only includes a `git mv` from the old table to the TableBase.tsx plz?
It will definitely help with history as well as making it easier to review this PS.